internal/cypress/sauce/runner.go

```diff
@@ -78,31 +86,48 @@ func (r *Runner) runSuites(fileID string) int {
 
 	// Collect results.
 	errCount := 0
+	completed := 0
+	total := len(r.Project.Suites)
+	inprogress := total
+
+	log.Info().Msgf("Suites completed: %d in progress: %d", completed, inprogress)
 	for i := 0; i < len(r.Project.Suites); i++ {
-		r := <-results
-		if r.err != nil {
-			log.Err(r.err).Str("suite", r.suiteName).Msg("Suite failed.")
+		res := <-results
+		completed++
+		inprogress--
+		logSuite(completed, inprogress, res.suiteName, res.browser, res.job.Passed)
+
+		if res.err != nil {
+			assetContent, err := r.JobReader.GetJobAssetFileContent(context.Background(), res.job.ID, resto.ConsoleLogAsset)
+			if err != nil {
+				log.Err(res.err).Str("suite", res.suiteName).Msg("Failed to get job asset.")
+			} else {
+				fmt.Println(string(assetContent))
```

Comment:
why switching to use `fmt.Print` here instead of `log.`?

Reply:
I wanted to display asset file content without this part `2:11PM INF`. We write to stdout the same as `fmt.Println`

Reply:
I fixed this and started using log.Info().Msg instead.